src/tools/rustfmt/src/attr.rs

```diff
@@ -17,6 +17,7 @@ use crate::shape::Shape;
 use crate::source_map::SpanUtils;
 use crate::types::{PathContext, rewrite_path};
 use crate::utils::{count_newlines, mk_sp};
+use rustc_ast::attr::AttributeExt;
```

Comment:
Could you move this import above with the other `rustc_ast` import? Same goes for the other rustfmt files that this PR touches.

Reply:
there ya go :)